.github/workflows/backup.yml

```diff
@@ -0,0 +1,31 @@
+on:
+  schedule:
+    - cron:  '0 */4 * * *'
```

Comment:
This makes the job run every four hours, not every day at 04:00. Is that really what you intended?
See [here](https://crontab.guru/#0_*/4_*_*_*).

Reloading the certificate in nginx is intended to happen every four hours, in case you used that as inspiration.

Reply:
Oh good that you found the cron bug! The value for the certificate renewal is the same, is that supposed to run every 4 hours or do we need to fix that as well?